mats/4.ms

```diff
@@ -3932,7 +3932,7 @@
    ;; ----------------------------------------
    ;; Check interaction of mutation and incremental generation promotion
 
-   (with-interrupts-disabled
+   (parameterize ([collect-request-handler void] [collect-maximum-generation (max (collect-maximum-generation) 2)])
```

Comment:
Alternate idea:
```
(or (< (collect-maximum-generation) 2)
    (with-interrupts-disabled ...))
```

Reply:
That was my first impulse, but this is the way it's handled in `measure-time` a few dozen lines higher in the file.  Either way is fine with me.

Reply:
Good find, then I now prefer your current implementation!